arelle/plugin/validate/NL/rules/nl_kvk.py

```diff
@@ -818,3 +820,30 @@ def rule_nl_kvk_3_6_3_3(
                   'Allowed characters include: A-Z, a-z, 0-9, underscore ( _ ), period ( . ), and hyphen ( - ). '
                   'Update filing naming to review unallowed characters. '
                   'Invalid filenames: %(invalidBasenames)s'))
+
+
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[
+        DISCLOSURE_SYSTEM_NL_INLINE_2024
+    ],
+)
+def rule_nl_kvk_6_1_1_1(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    NL-KVK.6.1.1.1: The size of the report package MUST NOT exceed 100 MB.
+    """
+    if val.modelXbrl.fileSource.fs: # must be a zip to be a report package
+        assert isinstance(val.modelXbrl.fileSource.fs, zipfile.ZipFile)
+        maxMB = float(MAX_REPORT_PACKAGE_SIZE_MBS)
+        _size = sum(zi.compress_size for zi in val.modelXbrl.fileSource.fs.infolist())
+        if _size > maxMB * 1048576:
```

Comment:
Is the limit 100 MB or 100 MiB (~105MB)?  The code is checking the latter.

Reply:
Documentation says MB, not MiB. I suppose it would be better to error on the side of smaller rather than the larger amount?